portfolio/src/main/webapp/index.html

```diff
@@ -13,11 +13,11 @@
   <body onload="getData()">
     <header>
         <div class="animation jumbotron">
-            <img src="images/Photo.jpeg" width="500" />
+            <img src="images/Photo.jpeg" alt="Photo" width="500">
```

Comment:
"Photo" isn't a super great description for those who can't see the image. Can you try describing the image?